src/main/java/nl/novi/Person.java

```diff
@@ -0,0 +1,180 @@
+package nl.novi;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Person implements IPerson {
```

Comment:
Netjes om hier een Interface te implementeren

Reply:
Hoi @RowanPl bedoel je dat het goed is dat ik hier interface heb gebruikt of bedoel je dat ik het hier moet implementeren. Ik heb hier namelijk al Interface gebruikt.